Examples/ComplexVaporExample/Sources/App/routes.swift

```diff
@@ -106,7 +102,7 @@ func routes(_ app: Application) throws {
             .unwrap(or: Abort(.internalServerError, reason: "Unexpectedly nil response from database"))
             .flatMapThrowing { result in
                 guard result.modifiedCount == 1 else {
```

Comment:
this check needs to be on `matchedCount`, since setting the favorite food to what it already is results in this cat-not-found error.

Reply:
nice catch!